src/test/java/uk/gov/hmcts/ccd/domain/types/DocumentValidatorTest.java

```diff
@@ -313,9 +308,7 @@ public void shouldFail_whenValidatingBinaryDocumentUrl() {
     }
 
     private DocumentValidator buildDocumentValidator(final String url) {
-        final ApplicationParams ap = mock(ApplicationParams.class);
-        when(ap.getValidDMDomain()).thenReturn(url);
-        return new DocumentValidator(ap);
+        return new DocumentValidator();
```

Comment:
We can use validator.validate(...) instead and remove this method

Reply:
That method is turning unused after the related reduntant tests are removed. Hence removed.